swagger.yaml

```diff
@@ -2521,6 +2523,22 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /supported_operators:
+    get:
+      tags:
+        - operators
+      operationId: ListSupportedOperators
+      description: Retrieves the list of supported operators
+      responses:
+        "200":
+          description: Success.
+          schema:
+            $ref: '#/definitions/list-operators'
```

Comment:
I think this should return a list of strings (e.g., ["lso", "ocs", "..."]?

We also wanted another API to get the properties JSON for each operator:
GET /supported-operators/{operator_type}

Reply:
fixed